.github/workflows/deploy.yml

```diff
@@ -2,8 +2,8 @@ name: "Deploy EVM Gateway to Cloud Run"
 
 on:
   push:
-    branches:
-      - main
+    # branches:
```

Comment:
Is this intentional? Do we want them to be commented-out?

Reply:
I'll revert these changes, they were only for testing